addon/components/frost-numeric-select.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Component definition for the numeric selection field component
+ */
+import layout from '../templates/components/frost-numeric-select'
+import Component from './frost-component'
+import Ember from 'ember'
+import {PropTypes} from 'ember-prop-types'
+import computed, {readOnly} from 'ember-computed-decorators'
+const {isPresent} = Ember
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+  layout,
+
+  // == PropTypes =============================================================
+
+  propTypes: {    // options
+    align: PropTypes.string,
+    isHookEmbedded: PropTypes.bool,
+    receivedHook: PropTypes.string,
+    tabindex: PropTypes.number,
+    type: PropTypes.string,
+    isArrowVisible: PropTypes.bool,
+    isAllowInputNegative: PropTypes.bool,
+    isAllowUseDownButtonToNegative: PropTypes.bool,
```

Comment:
So when the user is allowed to input negative number, they can also use the down button to get to the negative number?  And when the user is not allowed to input negative number, he can't use the down button to get them?